clients/admin-ui/src/pages/index.tsx

```diff
@@ -1,17 +1,61 @@
-import { Heading } from "@fidesui/react";
+import { Box, Button, Flex, Heading } from "@fidesui/react";
 import type { NextPage } from "next";
+import Head from "next/head";
+import NextLink from "next/link";
 import React from "react";
 
-import Layout from "~/features/common/Layout";
+import Header from "../features/common/Header";
+// import Setup from './setup';
```

Comment:
let's remove this

Reply:
This is there as a reminder, but I can remove 🙂